### ISSUE TITLE
DeepMind adaptation uses wrong length for patch names

### ISSUE THREAD

**Issue by Andy2No** [christofmuc/KnobKraft-orm#88]
I had a brief play with the DeepMind 12 implementation, recently, in 1.11.2, and found some problems.

The most obvious one was a square character, on the end of a lot of the patch names.  Another one was that if I exported a bank as separate sysex files, I got less than 128 patches, for some banks.

It turns out the patch name length is set as 18, in the adaptation, but it should be 16.  That seems to solve both of those problems... There may be some other problems left, but if so I'll start separate threads for those.

It looked like this (the square might be a different character, depending on system fonts used etc):

![Orm 1_11_2 DeepMind12 names bug](https://user-images.githubusercontent.com/8021302/109450065-ea760c80-7a41-11eb-80bb-ea8cfadd71e1.gif)

The problem was in nameFromDump(message);

```
def nameFromDump(message):
    nameBaseIndex = 223
    if isEditBufferDump(message):
        data = unescapeSysex(message[8:-1])
#        return ''.join([chr(x) for x in data[nameBaseIndex:nameBaseIndex+18]])
        return ''.join([chr(x) for x in data[nameBaseIndex:nameBaseIndex+16]])
    if isSingleProgramDump(message):
        data = unescapeSysex(message[10:-1])
#        return ''.join([chr(x) for x in data[nameBaseIndex:nameBaseIndex+18]])
        return ''.join([chr(x) for x in data[nameBaseIndex:nameBaseIndex+16]])
    return 'invalid'

```

I've attached the corrected .py file, in a zip.

[Deepmind_12.zip](https://github.com/christofmuc/KnobKraft-orm/files/6059068/Deepmind_12.zip)

**Comment by christofmuc** [comment]
@Andy2No Thank you! I will incorporate the fix into the next version. Interestingly, I though I had tested it with the factory patches downloadable from Behringer, but I might confuse it with another synth... Fix looks logical enough to me, it is good to hear the communication with the device works as expected!

**Comment by christofmuc** [comment]
Released with 1.12.0
